<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="quit_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
   <widget class="QPushButton" name="open_button">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>410</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>弹窗</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="display1">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>400</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSpinBox" name="display2">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>410</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="form1_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>界面1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="form2_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>界面2</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget1" native="true">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>640</width>
      <height>360</height>
     </rect>
    </property>
    <widget class="QLineEdit" name="value_1">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>140</y>
       <width>113</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="value_2">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>140</y>
       <width>113</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="value_sum">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>139</y>
       <width>113</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_1">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>141</y>
       <width>54</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>+</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>140</y>
       <width>54</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>=</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="calculate_button">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>240</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>计算</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget2" native="true">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>640</width>
      <height>360</height>
     </rect>
    </property>
    <widget class="QLCDNumber" name="sum_display">
     <property name="geometry">
      <rect>
       <x>133</x>
       <y>80</y>
       <width>281</width>
       <height>131</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Brush Script MT</family>
       <pointsize>48</pointsize>
       <italic>true</italic>
      </font>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="wel_widget" native="true">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>640</width>
      <height>360</height>
     </rect>
    </property>
    <widget class="QLabel" name="welcome">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>640</width>
       <height>360</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Welcome!</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <zorder>wel_widget</zorder>
   <zorder>widget1</zorder>
   <zorder>widget2</zorder>
   <zorder>quit_button</zorder>
   <zorder>open_button</zorder>
   <zorder>display1</zorder>
   <zorder>display2</zorder>
   <zorder>form1_button</zorder>
   <zorder>form2_button</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
